tensorflow/lite/micro/tools/make/ext_libs/person_detection_int8_vela_convert.sh

```diff
@@ -38,49 +38,40 @@ cd "${ROOT_DIR}"
 source tensorflow/lite/micro/tools/make/bash_helpers.sh
 
 DOWNLOADS_DIR=${1}
+GENERATED_SRCS_DIR=${2}
+
 if [ ! -d ${DOWNLOADS_DIR} ]; then
   echo "The top-level downloads directory: ${DOWNLOADS_DIR} does not exist."
   exit 1
 fi
 
-DOWNLOADED_PERSON_MODEL_INT8_PATH=${DOWNLOADS_DIR}/person_model_int8
-if [ ! -d ${DOWNLOADED_PERSON_MODEL_INT8_PATH} ]; then
-  echo "${DOWNLOADED_PERSON_MODEL_INT8_PATH} is not downloaded."
-  exit 1
-fi
-
 # Optimize downloaded model with Vela for Ethos-U.
 # See tensorflow/lite/micro/kernels/ethos_u/README.md for more info.
-CONVERTED_PERSON_MODEL_INT8=${DOWNLOADED_PERSON_MODEL_INT8_PATH}/person_detect_model_data_vela.cc
-if [ ! -f ${CONVERTED_PERSON_MODEL_INT8} ]; then
-  command xxd -v >&2 || (echo "xxd command is needed, please install.." && exit 1)
-  echo >&2 "Converting person detection int8 model to Ethos-U optimized model.."
-
-  # Convert original model to .tflite format.
-  grep -E "(0x[0-9a-f]{2}(,|))" ${DOWNLOADED_PERSON_MODEL_INT8_PATH}/person_detect_model_data.cc | xxd -r -p > \
-      ${DOWNLOADED_PERSON_MODEL_INT8_PATH}/person_detect.tflite
+MODEL_DIR=${GENERATED_SRCS_DIR}tensorflow/lite/micro/models
+CONVERTED_PERSON_MODEL_INT8=${MODEL_DIR}/person_detect_model_data_vela.cc
 
+if [ ! -f ${CONVERTED_PERSON_MODEL_INT8} ]; then
   # Compile an optimized .tflite version for Ethos-U.
   TEMPFILE=$(mktemp -d)/
   python3 -m venv $TEMPFILE
   source $TEMPFILE/bin/activate
   pip install --upgrade setuptools >&2
   pip install ethos-u-vela >&2
-  vela --accelerator-config=ethos-u55-256 ${DOWNLOADED_PERSON_MODEL_INT8_PATH}/person_detect.tflite \
-       --output-dir ${DOWNLOADED_PERSON_MODEL_INT8_PATH} >&2
+  vela --accelerator-config=ethos-u55-256 ${DOWNLOADS_DIR}/../../../models/person_detect.tflite \
+       --output-dir ${MODEL_DIR} >&2
   deactivate
 
   # Convert .tflite back to C array.
   echo "// This file is generated by $0." > ${CONVERTED_PERSON_MODEL_INT8}
-  echo '#include "tensorflow/lite/micro/tools/make/downloads/person_model_int8/person_detect_model_data.h"' >> \
+  echo '#include "tensorflow/lite/micro/models/person_detect_model_data.h"' >> \
        ${CONVERTED_PERSON_MODEL_INT8}
   echo -n "const " >> ${CONVERTED_PERSON_MODEL_INT8}
-  xxd -i ${DOWNLOADED_PERSON_MODEL_INT8_PATH}/person_detect_vela.tflite >> \
-      ${CONVERTED_PERSON_MODEL_INT8}
-  sed -i 's/tensorflow_lite_micro_tools_make_downloads_person_model_int8_person_detect_vela_tflite/g_person_detect_model_data/' \
+  xxd -i ${MODEL_DIR}/person_detect_vela.tflite >> ${CONVERTED_PERSON_MODEL_INT8}
+  sed -i 's/tensorflow_lite_micro_tools_make_gen_cortex_m_corstone_300_cortex_m55_default_genfiles_tensorflow_lite_micro_models_person_detect_vela_tflite/g_person_detect_model_data/' \
```

Comment:
nit: maybe comment on where 
tensorflow_lite_micro_tools_make_gen_cortex_m_corstone_300_cortex_m55_default_genfiles_tensorflow_lite_micro_models_person_detect_vela_tflite come from?

Reply:
yeah, I'm letting all this be for now since it is internals of the Vela system that I am not too familiar with.